Tools/jit/_targets.py

```diff
@@ -118,12 +118,20 @@ async def _compile(
             f"-I{CPYTHON / 'Python'}",
             "-O3",
             "-c",
+            # This debug info isn't necessary, and bloats out the JIT'ed code.
+            # We *may* be able to re-enable this, process it, and JIT it for a
+            # nicer debugging experience... but that needs a lot more research:
             "-fno-asynchronous-unwind-tables",
+            # Don't call built-in functions that we can't find or patch:
             "-fno-builtin",
-            # SET_FUNCTION_ATTRIBUTE on 32-bit Windows debug builds:
+            # This breaks SET_FUNCTION_ATTRIBUTE on 32-bit Windows debug builds.
+            # It's not clear why:
```

Comment:
Because it's the only tier 2 instruction in bytecodes.c that uses a `switch` statement? Possibly it's better to state that it breaks jitting `switch` statements?

Reply:
Ah, of course! Nice catch.

Reply:
Would it be to better replace the `switch` with a nested `if` statement?

Reply:
Alternatively, change the flags for the various `MAKE_FUNCTION_...` from 1,2,4,8 to 0,1,2,3 and
then prefix `SET_FUNCTION_ATTRIBUTE` with `replicate(4)`.

Reply:
Given that `SET_FUNCTION_ATTRIBUTE` should almost never be jitted, the nested if solution is probably the best for performance and memory use.

Reply:
I just fixed the bug in our COFF parsing instead. :)